src/components/Layouts/Stack.css

```diff
@@ -0,0 +1,3 @@
+.wui-Stack {
+  display: flex;
+}
```

Comment:
Please add an empty line at the end of each file 👍

Reply:
I would prefer to rename the file to `stack.css` instead. But it is just based in my previous projects... there is no other reason.